Docs/user_guide/examples.rst

```diff
@@ -73,40 +80,106 @@ Running the notebooks
 
 To run the notebooks, follow the instructions below.
 
-Prerequisites
--------------
+1. Run the notebook server
+--------------------------
+
+1. Install the Jupyter metapackage using the following command. 
+   (Prepend the command with ``sudo -H`` if necessary to grant admin privilege.)
+
+   .. code-block:: shell
+
+      python3 -m pip install jupyter
 
-#. Install the Jupyter metapackage using the following command. (If necessary, pre-pend the command with "sudo -H".)
-  .. code-block:: shell
+2. Start the notebook server as follows:
 
-    python3 -m pip install jupyter
-#. Start the notebook server as follows:
-.. code-block:: shell
+   .. code-block:: shell
     
-    jupyter notebook --ip=* --no-browser &
-#. The command generates and displays a URL in the terminal. Copy and paste the URL into your browser.
-#. Install AIMET and its dependencies using the instructions in :doc:`install`.
+      jupyter notebook --ip=* --no-browser &
 
+   The command generates and displays a URL in the terminal. 
+   
+3. Copy and paste the URL into your browser.
 
-1. Download the example notebooks and related code
+4. Install AIMET and its dependencies using the instructions in :doc:`install`.
+
+
+2. Download the example notebooks and related code
 --------------------------------------------------
 
-#. Clone the AIMET repo by running the following commands:
-.. code-block:: shell
-   WORKSPACE="<absolute_path_to_workspace>"
-   mkdir $WORKSPACE && cd $WORKSPACE
-   # Identify the release tag (<release_tag>) of the AIMET package that you're working with at: https://github.com/quic/aimet/releases.
-   git clone https://github.com/quic/aimet.git --branch <release_tag>
-   # Update the path environment variable:
-   export PYTHONPATH=$PYTHONPATH:${WORKSPACE}/aimet
-#. The dataloader, evaluator, and trainer used in the examples is for the ImageNet dataset.
-  Download the ImageNet dataset from here: https://www.image-net.org/download.php
-
-2. Run the notebooks
+Set up your workspace using the following steps:
+
+1. Set a workspace path:
+
+   .. code-block:: shell
+      
+      WORKSPACE="<absolute_path_to_workspace>"
+
+2. Create and move to the workspace:
+
+   .. code-block:: shell
+    
+      mkdir $WORKSPACE && cd $WORKSPACE
+
+3. Identify the release tag (``<release_tag>``) of the AIMET package that you're working with at: https://github.com/quic/aimet/releases.
+<<<<<<< HEAD
```

Comment:
Could this marker have been unintentionally added while resolving a merge conflict?

Reply:
Yes. See below.